subsys/bluetooth/host/Kconfig

```
@@ -1163,6 +1163,15 @@ config BT_CONN_DISABLE_SECURITY
 	  WARNING: This option enables anyone to snoop on-air traffic.
 	  Use of this feature in production is strongly discouraged.
 
+config BT_SMP_ENFORCE_LEGACY_PAIRING
```

Should it be


```suggestion
config BT_SMP_LEGACY_PAIR_ONLY
```

to following the naming of 
```
config BT_SMP_OOB_LEGACY_PAIR_ONLY
config BT_SMP_SC_ONLY
config BT_SMP_SC_PAIR_ONLY
```